training/src/main/java/wolox/training/models/Book.java

```diff
@@ -11,12 +12,19 @@
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
 import javax.persistence.ManyToMany;
+import org.thymeleaf.util.StringUtils;
 import wolox.training.exceptions.BookNotOwnedByUserException;
 
 @Entity
 @ApiModel(description = "Books from the TrainingApi")
 public class Book {
 
+    private static final String NULL_PARAMETER = "The parameter should not be null.";
+
+    private static final String EMPTY_STRING = "String should have at least 1 character.";
+
+    private static final String NEGATIVE_QUANTITY = "The quantity should be a positive integer.";
```

Comment:
Knowing that some of this constants are used in more than one class, maybe you can have a class MessageConstants.java with a private constructor (Because you'll only be using the static fields) in an extra package named utils
Each constant would be public static final, and then you can access them like MessageConstants.NULL_PARAMETER

Reply:
suggestion applied :)

Reply:
Great! Just one more comment: Usually you don't want this kind of classes (Like MessageConstants) to be instantiated because you can use all of that in a static way. Java adds a default constructor per class, so to avoid the instantiation you can override the default constructor adding a private constructor:
```
private MessageConstants(){
    //This constructor is defined to avoid the instantiation of this class
}
```

Reply:
I would add final to the class since we don't need to extend anything from it and it allows more clarity in how we encapsulate our objects.

Reply:
Final and private constructor added.